koku/api/report/serializers.py

```diff
@@ -105,31 +135,42 @@ class BaseSerializer(serializers.Serializer):
     """A common serializer base for all of our serializers."""
 
     _opfields = None
+    _tagkey_support = None
```

Comment:
This flag is a replacement for the `tag_fields` attribute that was being set previously. `_init_tagged_fields()` looks for classes with `_tagkey_support = True` and only adds the tagkey attributes to classes that support tags.